Unable to build `container-build` / `build-image` on macOS

**What happened:**
Running `make kvm-dev` from `main` results in:
```
gyptazy@knusperbox gardenlinux % git checkout main
Switched to branch 'main'
Your branch is up to date with 'origin/main'.

gyptazy@knusperbox gardenlinux % make kvm-dev     
make --directory=container build-base-test
WARNING: You are still using a debian:testing-slim as a temporary replacement of gardenlinux/slim:latest!

Please run 'make slim' to fix
mkdir -p base-test/_pipfiles
cp ../tests/Pipfile* base-test/_pipfiles
[+] Building 0.1s (11/11) FINISHED                                                                                                                                                   
 => [internal] load build definition from Dockerfile                                                                                                                            0.0s
 => => transferring dockerfile: 37B                                                                                                                                             0.0s
 => [internal] load .dockerignore                                                                                                                                               0.0s
 => => transferring context: 2B                                                                                                                                                 0.0s
 => [internal] load metadata for docker.io/gardenlinux/slim:latest                                                                                                              0.0s
 => [internal] load build context                                                                                                                                               0.0s
 => => transferring context: 99.50kB                                                                                                                                            0.0s
 => [1/6] FROM docker.io/gardenlinux/slim                                                                                                                                       0.0s
 => CACHED [2/6] RUN echo "deb http://deb.debian.org/debian testing main contrib non-free" > /etc/apt/sources.list      && apt-get update      && apt-get install -y --no-inst  0.0s
 => CACHED [3/6] RUN pip install pipenv==2022.4.8                                                                                                                               0.0s
 => CACHED [4/6] COPY _pipfiles /_pipfiles                                                                                                                                      0.0s
 => CACHED [5/6] RUN cd /_pipfiles && pipenv --python 3.10 install --system --dev && cd / && rm -rf /_pipfiles                                                                  0.0s
 => CACHED [6/6] WORKDIR /gardenlinux/tests                                                                                                                                     0.0s
 => exporting to image                                                                                                                                                          0.0s
 => => exporting layers                                                                                                                                                         0.0s
 => => writing image sha256:87a44eb6abfdc3247015dbf741ec458e772c104d7f459655a7033c9caac0101a                                                                                    0.0s
 => => naming to docker.io/gardenlinux/base-test:today                                                                                                                          0.0s

Use 'docker scan' to run Snyk tests against images to find vulnerabilities and learn how to fix them
make --directory=container build-image
WARNING: You are still using a debian:testing-slim as a temporary replacement of gardenlinux/slim:latest!

Please run 'make slim' to fix
cp ../bin/garden-feat.go build-image/
if [ ! -n "$(docker image ls gardenlinux/build-image:`../bin/garden-version --major | tr -d '\n'; echo -n "."; ../bin/garden-version --minor` --format "{{.Repository}}:{{.Tag}}")" ]; then \
		docker image rm --force $(docker image ls gardenlinux/build-image --format "{{.Repository}}:{{.Tag}}") || true; \
	fi
"docker image ls" requires at most 1 argument.
See 'docker image ls --help'.

Usage:  docker image ls [OPTIONS] [REPOSITORY[:TAG]]

List images
Untagged: gardenlinux/build-image:today
Deleted: sha256:32d964dab7f84e079f0557e6a32b868fcdaa86583b9b588110e97007634bd54d
"docker build" requires exactly 1 argument.
See 'docker build --help'.

Usage:  docker build [OPTIONS] PATH | URL | -

Build an image from a Dockerfile
make[1]: *** [build-image] Error 1
make: *** [container-build] Error 2
```

**How to reproduce it**:
- Checkout `main`
- Run `make kvm-dev`
  
**Environment:**
Reproducible with `Docker` on `macOS`.

| Name | Value |
| ------------------- | ----- |
| Garden Linux Branch | *main* |
| Container runtime (version)  | *Docker (4.9.0)* | 
| Host OS of build machine  | *macOS* |

--- a/container/Makefile
+++ b/container/Makefile
@@ -1,5 +1,7 @@
 VERSION=`../bin/garden-version`
-VERSION_NUMBER=`../bin/garden-version --major | tr -d '\n'; echo -n "."; ../bin/garden-version --minor`
+VERSION_NUMBER_MAJOR=$(shell ../bin/garden-version --major)
+VERSION_NUMBER_MINOR=$(shell ../bin/garden-version --minor)
+VERSION_NUMBER=$(VERSION_NUMBER_MAJOR).$(VERSION_NUMBER_MINOR)
 ALTNAME=
 ALTNAME_INTERNAL=$(shell [ -n "$(ALTNAME)" ] && printf "%s %s" "-t" "$(ALTNAME)" )
 

